active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,199 @@
+| SelectedPageIndex | The 0 based index that is currently selected. It will default to the first index. | 0
+| MaxVisualIndicators | The maximum number of pips that will appear in the control at once. If the NumberOfPages is greater than this value, the UI will scroll so that the selected pip is centered. | 5
+| Orientation | The orientation of the control. Can be either Vertical or Horizontal. | Horizontal
+| PipsPagerSelectedIndexChangedEvent | Event that is fired after the user selects a pip or the directional buttons. This event will return the index number of the page that the user selected. | N/A
+*Command for the buttons?
+
+# API Details
+```IDL
+enum PipsPagerButtonVisibility
+{
+  Visible,
+  VisibleOnHover,
+  Collapsed, 
+}
+
+runtimeclass PipsPagerSelectedIndexChangedEventArgs
+{
+    Int32 NewPageIndex{get; };
+    Int32 PreviousPageIndex{get; };
+};
+
+unsealed runtimeclass PipsPagerTemplateSettings : Windows.UI.Xaml.DependencyObject
+{
+    Windows.Foundation.Collections.IVector<Object> PipsPagerItems { get; }; 
+}
+
+
+unsealed runtimeclass PipsPager : Windows.UI.Xaml.Controls.Control
+{
+    PipsPager();
+
+    [MUX_DEFAULT_VALUE("0")]
+    Int32 NumberOfPages;
+
+    [MUX_DEFAULT_VALUE("0")]
+    Int32 SelectedPageIndex;
+
+    [MUX_DEFAULT_VALUE("5")]
+    Int32 MaxVisualIndicators;
+
+    [MUX_DEFAULT_VALUE("Horizontal")]
+    Windows.UI.Xaml.Controls.Orientation Orientation;
+
+    [MUX_DEFAULT_VALUE("Collapsed")]
+    PagerControlButtonVisibility PreviousButtonVisibility;    
+    [MUX_DEFAULT_VALUE("Collapsed")]
+    PagerControlButtonVisibility NextButtonVisibility;
+
+    Windows.UI.Xaml.Style PreviousButtonStyle;
+    Windows.UI.Xaml.Style NextButtonStyle; 
+
+    Windows.UI.Xaml.Style SelectedIndicatorButtonStyle;
+    Windows.UI.Xaml.Style DefaultIndicatorButtonStyle;
+
+
+    event Windows.Foundation.TypedEventHandler<PipsPager, PipsPagerSelectedIndexChangedEventArgs> PipsPagerSelectedIndexChanged;
+
+    PipsPagerTemplateSettings TemplateSettings{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty NumberOfPagesProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedPageIndexProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxVisualIndicatorsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty OrientationProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonVisibilityProperty { get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonVisibilityProperty { get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedIndicatorButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty DefaultIndicatorButtonStyleProperty{ get; };
+}
+```
+
+![](images/pager-control-vertical-pips-scroll-ex.jpg)
+
+## Theme Resources
```

Comment:
Do we have any precedent on docs.microsoft.com where we've done a good job listing theme resources, to explain what they're about? This is a useful list but doesn't make much sense to most people.

Reply:
See [Button](https://github.com/microsoft/microsoft-ui-xaml-specs/pull/104) API example.